sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -881,10 +881,10 @@ object DateTimeUtils {
    * Returns number of months between time1 and time2. time1 and time2 are expressed in
    * microseconds since 1.1.1970.
    *
-   * If time1 and time2 having the same day of month, or both are the last day of month,
-   * it returns an integer (time under a day will be ignored).
+   * If time1 and time2 are on the same day of month, or both are the last day of month,
+   * returns an integer (time under a day will be ignored).
```

Comment:
It seems a bit awkward because it actually returns a double. Shall we fix this like .. `returns an integer (time under a day will be ignored)` -> `time under a day will be ignored.`?